assets/js/util/test/prepareForReadableLargeNumber.js

```diff
@@ -0,0 +1,105 @@
+/**
+ * Internal dependencies
+ */
+import { prepareForReadableLargeNumber } from '../';
+
+const numbersBelowOneThousand = [
+	[
+		1,
+		1,
+	],
+	[
+		500,
+		500,
+	],
+	[
+		999,
+		999,
+	],
+];
+
+const numbersBetweet1000And10000 = [
+	[
+		1000,
+		1,
+	],
+	[
+		1201,
+		1.2,
+	],
+	[
+		1500,
+		1.5,
+	],
+	[
+		1999,
+		2,
+	],
+	[
+		6566,
+		6.6,
+	],
+];
+
+const numbersBetween10000And1000000 = [
+	[
+		10000,
+		10,
+	],
+	[
+		10100,
+		10,
+	],
+	[
+		10500,
+		11,
+	],
+];
+
+const numbersOver1000000 = [
```

Comment:
Can these be inlined inside the `describe()` block above their respective `it.each`? It will make it a bit easier to digest the test and comprehend both the test inputs, outputs, and logic at once. I appreciate the convenience of writing tests with it.each (it's very DRY), but the aim of our test code should be comprehension over all else, even if it means repeating ourselves a bit. Tests like these can be difficult to come back to and understand because the flow of the test and the inputs/outputs of the test are not coupled.

Placing these closer to their tests should help with that a bit—I don't think we have to write a test for each number manually, but the proximity should help link them together cognitively 😄

Reply:
(Note: this comment applies to all the tests here; anyplace we can get the inputs/results closer to the actual test case code I'm for it 😄)